### ISSUE TITLE
Map not showing up with 1.13.1 worlds

### ISSUE THREAD

**Issue by chland** [Querz/mcaselector#2]
I'm running mcaselector on a Win 10 machine with the following Java-version:

```
java version "1.8.0_161"(should be "JRE 8u161-b12")
Java(TM) SE Runtime Environment (build 1.8.0_161-b12)
```

If i open a 1.12 world, everything is fine but if i open a 1.13.1 world the screen looks like this:

![image](https://user-images.githubusercontent.com/472894/45747435-22888c80-bc06-11e8-96be-e481cee38f93.png)

This is from a test-world that contains 4 region-files. It looks like mcaselector can detect the region-files just fine but can't load/display the chunk-data.

I took a look at the region-files using NBTExplorer and they are using DataVersion 1628.

**Comment by chland** [comment]
Oh, andI totally forgot to add: when i run it in debug-mode it throws a ton of casting-errors:

```
java.lang.ClassCastException: Cannot cast net.querz.nbt.IntArrayTag to net.querz.nbt.ByteArrayTag
        at java.lang.Class.cast(Unknown Source)
        at net.querz.nbt.CompoundTag.get(CompoundTag.java:71)
        at net.querz.nbt.CompoundTag.getByteArrayTag(CompoundTag.java:109)
        at net.querz.nbt.CompoundTag.getByteArray(CompoundTag.java:169)
        at net.querz.mcaselector.version.anvil113.Anvil113ChunkDataProcessor.drawChunk(Anvil113ChunkDataProcessor.java:24)
        at net.querz.mcaselector.io.MCAChunkData.drawImage(MCAChunkData.java:117)
        at net.querz.mcaselector.io.MCAFile.createImage(MCAFile.java:259)
        at net.querz.mcaselector.tiles.Tile.generateImage(Tile.java:280)
        at net.querz.mcaselector.io.RegionImageGenerator$MCAImageProcessJob.execute(RegionImageGenerator.java:83)
        at net.querz.mcaselector.io.ProcessDataJob.run(ProcessDataJob.java:21)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
```

**Comment by Querz** [comment]
Thanks for pointing this out.
Apparently i only tested 1.13 worlds while they were still in the snapshot phase, as this tool was originally intended to crop worlds while migrating them from 1.12 to 1.13.

It's fixed in [1.3.1](https://github.com/Querz/mcaselector/releases/tag/1.3.1).

**Comment by chland** [comment]
Thanks for fixing the issue so quickly!
